### DISCUSSION TITLE
Tests translation rules vague around float formatting

### DISCUSSION THREAD

**Discussion by Patitotective** [kdl-org/kdl#297] [General]
I'm making a KDL parser for the Nim programming language but the [translation rules](https://github.com/kdl-org/kdl/tree/main/tests#translation-rules) for numbers in the test suite doesn't specify digits precision for the scientific notation.

**Comment by larsgw** [comment]
I would assume the same level of precision as in the input (i.e. the same float, but with the point fixed), but I don't know for sure and either way that could be included in the text.
